docker-images/kafka.version

```diff
@@ -0,0 +1,3 @@
+2.7.0
+2.7.1
+2.8.0
```

Comment:
Why keep old versions such as 2.7.0 or 2.7.1 which we are dropping from Strimzi and not add 2.8.1 or 3.0.0 instead?

Reply:
We discussed this with @Frawless and we wanted to support same versions as we are supporting in Strimzi ATM -> before the PR with removal of 2.7.0 and 2.7.1 will be merged. I will change it to `2.8.0`,`2.8.1` and `3.0.0`